tests/MongoDB.Driver.Core.TestHelpers/CoreExceptionHelper.cs

```diff
@@ -100,6 +100,9 @@ public static Exception CreateException(string errorType)
                 case "TimedOutSocketException":
                     return new SocketException((int)SocketError.TimedOut);
 
+                case "ConnectionRefusedSocketException":
+                    return new SocketException((int)SocketError.ConnectionRefused);
+
```

Comment:
Is this related to this ticket?

Reply:
it's  related to this test: https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/157/files#diff-0537777d32b586e768ac1e74c10677b06f8933c41fba72c3f11a6f74862925e7R547 (CustomEndpointTest)